packages/core-subscriptions/db/subscriptions/schema.js

```diff
@@ -64,3 +64,7 @@ export const Schema = new SimpleSchema(
 );
 
 Subscriptions.attachSchema(Schema);
+
+Subscriptions.rawCollection().createIndex({ userId: 1 });
```

Comment:
Please create the indexes with an "init" function like in other packages (core-warehousing for ex.) so we can control the order of execution and it's consistent